code/datums/elements/cleaning.dm

```diff
@@ -14,27 +14,17 @@
 	if(!isturf(tile))
 		return
 
-	SEND_SIGNAL(tile, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_STRENGTH_BLOOD)
-	for(var/A in tile)
-		if(is_cleanable(A))
-			qdel(A)
-		else if(istype(A, /obj/item))
+	tile.wash(CLEAN_WASH)
+	for(var/atom/A in tile)
+		// Clean small items that are lying on the ground
+		if(istype(A, /obj/item))
```

Comment:
```suggestion
	for(var/am in tile)
		// Clean small items that are lying on the ground
		if(isitem(am))
```
There can only be movable atoms in tiles. Also, no need to typecheck on the loop.